tldr.ts

```diff
@@ -1,53 +1,31 @@
-export async function fetchTLDR(API_KEY) {
-	const headers = {
-		"Content-Type": "application/json",
-	};
+/**
+ * tldr creates a "TLDR" of a Discord message.
+ */
+export async function tldr(options: TLDROptions) {
+  const text =
+    `Create a "TLDR" of this Discord mesage: ${options.author}: ${options.message}\n\nTL;DR: `;
+  const response = await fetch(
+    "https://generativelanguage.googleapis.com/v1beta2/models/text-bison-001:generateText?key=" +
+      options.apiKey,
+    {
+      method: "POST",
+      headers: new Headers([
+        ["Content-Type", "application/json"],
+      ]),
+      body: JSON.stringify({ prompt: { text } }),
```

Comment:
Future work: We should ensure that the request complies with the [`GenerateTextRequest`](https://github.com/googleapis/googleapis/blob/77c99e43177c76ae1c1edacee7b6ac4e35a42f3d/google/ai/generativelanguage/v1beta2/text_service.proto#L59) object satisfied.
```suggestion
      body: JSON.stringify({ prompt: { text } } satisfies GenerateTextRequest),
```